docs/config/processors/versions-data.js

```diff
@@ -12,23 +15,93 @@ module.exports = function generateVersionDocProcessor(gitData) {
   return {
     $runAfter: ['generatePagesDataProcessor'],
     $runBefore: ['rendering-docs'],
+    blacklist: ['1.3.4-build.3588'],
     $process: function(docs) {
 
-      var versionDoc = {
+      var blacklist = this.blacklist;
```

Comment:
OK. The main reason was that we appear to have npm published a random rogue release, 1.3.4-build.3588, which does not have an associated folder on code.angularjs.org